src/budy/models/voucher_use.py

```
@@ -0,0 +1,96 @@
+
+import appier
+
+from . import base
+
+
+class VoucherUse(base.BudyBase):
+
+    USAGE_TYPE_S = dict(
+        percentage="percentage",
+        value="value",
+    )
```

Pylint (3.3.7)

[refactor] Consider using '{"percentage": 'percentage', "value": 'value'}' instead of a call to 'dict'.

(R1735)